packages/tasks/src/local-apps.ts

```diff
@@ -250,6 +250,13 @@ export const LOCAL_APPS = {
 		displayOnModelPage: (model) => model.library_name === "diffusers" && model.pipeline_tag === "text-to-image",
 		deeplink: (model) => new URL(`https://models.invoke.ai/huggingface/${model.id}`),
 	},
+	gpt4all: {
+		prettyLabel: "GPT4All",
+		docsUrl: "https://nomic.ai/gpt4all",
+		mainTask: "text-generation",
+		displayOnModelPage: isGgufModel,
+		deeplink: (model) => new URL(`gpt4all://models/huggingface/${model.id}`),
```

Comment:
Is the deeplink URL deployed? It didn't work for me on v3.2.1

Reply:
Deep link support is not deployed out but will work with that path. Happy to remove it for now and add deeplink in a subsequent PR if you don't want the preemptive path.